static/js/loading.js

```diff
@@ -46,7 +46,7 @@ exports.make_indicator = function (outer_container, opts) {
     // These width calculations are tied to the spinner width and
     // margins defined via CSS
     container.css({width: 38 + text_width,
-                   height: 38});
+                   height: 0});
```

Comment:
This is change is to remove the bouncy behavior in the settings panel. Kindly Update me if there is any other better method.

Reply:
@Balaji2198 I think maybe we should make this an argument, since I'm worried this will break other uses of the spinner?